data/json/items/gunmod/rail.json

```diff
@@ -95,5 +95,24 @@
     "mod_targets": [ "smg", "rifle", "pistol", "shotgun", "crossbow", "launcher" ],
     "flags": [ "LASER_SIGHT" ],
     "min_skills": [ [ "weapon", 2 ], [ "gun", 1 ] ]
+  },
+  {
+    "id": "offset_grip",
+    "type": "GUNMOD",
+    "name": { "str": "offset grip" },
+    "description": "An angled grip set at 45° that attaches to a side rail, allowing for a grip while leaving the underbarrel rail free, or on guns that have integral bipods. Also allows for slightly greater control with smaller movements.",
+    "weight": "125 g",
+    "volume": "250 ml",
+    "price": "300 USD",
+    "price_postapoc": "7 USD 50 cent",
+    "install_time": "2 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "rail",
+    "handling_modifier": 2,
```

Comment:
I'd change the handling modifier to 6 and remove the aimspeed modifier. I'm unsure if foregrips of any kind actually make you aim any faster. Also, wouldn't these be able to be attached to the main underbarrel location as well? I'd also give it a "blacklist_mod" and put in all the other foregrips, just to prevent someone slapping on a bunch of extra foregrips to decrease recoil.

Reply:
Do you have some information on the IRL use of them? All I've been able to find is just airsoft stuff sadly

Reply:
Ironically that's all I can seem to find as well, though they do make some good points in this post I found [here](https://www.reddit.com/r/airsoft/comments/14cwpe1/is_there_any_advantage_at_putting_the_front_grip/)
It's partially a comfort thing, but some guns' magazines also have issues with straight vertical grips, and having your grip at an angle keeps you from having clearance issues with things like drum magazines, box magazines, or banana mags.

Reply:
I am skeptical of having a gun mod that only seems to exist in airsoft